plugins/node-restart.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: node-restart
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/MnrGreg/kubectl-node-restart/releases/download/v0.0.2/v0.0.2.zip
+    sha256: "44db0eee9603085652837d26c5352287864f158b557f78ae19325724640535f8"
+    files:
+    - from: "*.sh"
+      to:  "."
+    - from: "LICENSE"
+      to: "."
+    bin: "node-restart.sh"
+  shortDescription: >-
+    Restart Kubernetes Nodes sequentially & gracefully
```

```suggestion
    Restart cluster nodes sequentially & gracefully
```
Kubernetes is not necessary, because this is `kubectl` :)

---

revised